pydynamo/storage/disk/sstable.py

```diff
@@ -0,0 +1,91 @@
+from ..store import Store
+from ..memory.inmemorystore import InMemoryStore
+from ..error import StorageException
+from ..error import ErrorType
+from datetime import datetime
+from .sstableiterator import SStableIterator
+
+
+class SSTable(Store):
+    def __init__(self, store_name: str, store_id: int,
+                 mem_table: InMemoryStore, percentage: float) -> None:
+        self.store_name = store_name
+        self.sstable_file_name = store_name + str(store_id) + ".db"
+        self.index_file_name = store_name + str(store_id) + ".index"
+        self.mem_table = mem_table
+        self.percentage = percentage
+        # index_table:
+        # row: (key, offset)
+        self.index_table: list = []
+        self.size = 0
+        self._flush_to_disk()
+        self.cur = None
+
+    def get(self, key: str) -> str:
+        iterator = self.iterator()
+        iterator.seek(key)
+        return iterator.value()
+
+    def get_timestamp(self, key: str) -> str:
+        iterator = self.iterator()
+        iterator.seek(key)
+        return iterator.timestamp()
+
+    def get_row_length(self, key: str) -> int:
+        iterator = self.iterator()
+        iterator.seek(key)
+        return iterator.length()
+
+    def set(self, key: str, value: str):
+        raise StorageException(ErrorType.NOT_FOUND,
+                               "SSTable cannot be modified.")
+
+    def iterator(self):
+        return SStableIterator(self.sstable_file_name,
+                               self.index_file_name,
+                               self.index_table, self.size)
+
+    def remove(self, key: str) -> None:
+        raise StorageException(ErrorType.NOT_FOUND,
+                               "SSTable cannot be modified.")
+
+    def _flush_to_disk(self):
+        iterator = self.mem_table.iterator()
+        sstable_file = open(self.sstable_file_name, "w")
+        index_file = open(self.index_file_name, "w")
+        offset = 0
+        key_index = 0
+        interval = self._index_interval(self.mem_table.get_size())
+        while iterator.valid():
+            iterator.next()
+            key = iterator.key()
+            value = iterator.value()
+            # add tuple(key, offset) to index_table in memory
+            if key_index % interval == 0:
+                self.index_table.append((key, bin(offset).zfill(32)))
+            # store row into index_file and sstable_file
+            key_size = len(key)
+            val_size = len(value)
+            # index_file:
+            # row: key_size(32) + key + offset(32)
+            index_file.write(bin(key_size).zfill(32))
```

Comment:
index_file.write(bin(key_size).zfill(32)) this is not correct;  consider using https://docs.python.org/2/library/struct.html